frontend/src/app/pages/Profile/index.js

```diff
@@ -18,17 +18,30 @@ class ProfileComponent extends Component {
 
     const {
       user: { name, rating, id },
-      statistics: { totalMatches, winRatio, longestStreak, matchChanges },
+      statistics: {
+        totalMatches,
+        winRatio,
+        matchChanges,
+        streaks: {
+          currentWinStreak,
+          longestWinStreak,
+          currentEloGainStreak,
+          highestEloGainStreak,
+        },
+      },
     } = this.props
 
     return (
       <Box Display="inline-block" Padding="20px" Margin="20px 0">
         <Title>{name}</Title>
-        <Subtitle>Elo rating: {rating}</Subtitle>
+        <Subtitle>ELO rating: {rating}</Subtitle>
```

Comment:
I think it's someone's name, it was spelled 'Elo' on purpose 😄

Reply:
Oops, ok, I'll fix that.

Reply:
[Arpad Elo](https://en.wikipedia.org/wiki/Arpad_Elo)